styles/Homepage/Homepage.module.css

```diff
@@ -0,0 +1,245 @@
+.container {
```

Comment:
Not a blocker, but a potential refactoring. It looks like most all styles are in this Homepage.module.css. Preferably there would be more separation between your CSS styles. Especially since not all of them are used on the homepage.

Reply:
I split out css now wherever possible.